.eslintrc.js

```diff
@@ -16,6 +16,9 @@ module.exports = {
 		'id-length': 'off',
 		'import/no-extraneous-dependencies': 'off',
 		'jsx-a11y/click-events-have-key-events': 'off',
+		'@typescript-eslint/no-unsafe-call': 'off',
```

Comment:
We also disable this in the VIP dashboard, making it impossible to pass tests using user events.

Reply:
When you replaced `fireEvent` with `userEvent` (i.e. in the Accordion test), the test would fail unless this rule was disabled? I haven't been able to replicate the error so I'm curious if I'm missing something here.

Reply:
Thanks for the context!